src/Raven.Client/Documents/Operations/OngoingTasks/ToggleOngoingTaskStateOperation.cs

```diff
@@ -56,6 +56,7 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
             }
 
             public override bool IsReadRequest => false;
+            public override bool IsClusterCommand => true;
```

Comment:
I wonder if it wouldn't be more appropriate to handle this via a marker interface, instead of a property.

Reply:
The reasons I did it as a property is:
1. Not to forget any command
2. Enforce future command not to forget it :-)

Reply:
1) That required to change a lot of unrelated code.
2) Can do that more easily by validating the commands that go to the cluster directly.